atd-etl/Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN apk add bash build-base python3-dev libffi-dev openssl-dev \
 #RUN apk update && apk add build-base python3-dev libffi-dev openssl-dev && pip3 install magic-wormhole
 
 RUN pip install cryptography==3.3.2 splinter selenium requests \
-    awscli web-pdb sodapy boto3 mail-parser
+    awscli web-pdb sodapy boto3 mail-parser python-magic==0.4.27
```

Comment:
might as well pin the rest of these or take the plunge for a `requirements.txt` ?

Reply:
Yeah, this is making me think about [this](https://github.com/cityofaustin/atd-vz-data/pull/1150#discussion_r1097734519) that I just saw. Maybe worth a short dev sync discussion so we handle these the same way.